packages/components/descriptions/src/descriptions-cell.ts

```diff
@@ -54,65 +59,74 @@ export default defineComponent({
 
     switch (this.type) {
       case 'label':
-        return h(
-          this.tag,
-          {
-            style,
-            class: [
-              ns.e('cell'),
-              ns.e('label'),
-              ns.is('bordered-label', border),
-              ns.is('vertical-label', isVertical),
-              labelAlign,
-              labelClassName,
-            ],
-            colSpan: isVertical ? span : 1,
-          },
-          label
+        return withDirectives(
+          h(
+            this.tag,
+            {
+              style,
+              class: [
+                ns.e('cell'),
+                ns.e('label'),
+                ns.is('bordered-label', border),
+                ns.is('vertical-label', isVertical),
+                labelAlign,
+                labelClassName,
+              ],
+              colSpan: isVertical ? span : 1,
+            },
+            label
+          ),
+          directives
         )
       case 'content':
-        return h(
-          this.tag,
-          {
-            style,
-            class: [
-              ns.e('cell'),
-              ns.e('content'),
-              ns.is('bordered-content', border),
-              ns.is('vertical-content', isVertical),
-              align,
-              className,
-            ],
-            colSpan: isVertical ? span : span * 2 - 1,
-          },
-          content
+        return withDirectives(
+          h(
+            this.tag,
+            {
+              style,
+              class: [
+                ns.e('cell'),
+                ns.e('content'),
+                ns.is('bordered-content', border),
+                ns.is('vertical-content', isVertical),
+                align,
+                className,
+              ],
+              colSpan: isVertical ? span : span * 2 - 1,
+            },
+            content
+          ),
+          directives
```

Comment:
I don’t understand what you mean. Are you saying that the switch can satisfy both `label` and `content` at the same time? 🤔

Reply:
[demp](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnO1xuXG5jb25zdCB2Rm9jdXMgPSB7IG1vdW50ZWQ6IChlbCkgPT4ge1xuICBjb25zb2xlLmxvZygndkZvY3VzJywgZWwpO1xuICBlbC5mb2N1cygpXG59IH1cblxuY29uc3QgdlRlc3QgPSAoZWwpID0+IHtcbiAgY29uc29sZS5sb2coJ3ZUZXN0JywgZWwpXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZGl2IHYtdGVzdD5mb288L2Rpdj5cbiAgPEVsRGVzY3JpcHRpb25zIGJvcmRlciBkaXJlY3Rpb249XCJ2ZXJ0aWNhbFwiPlxuICAgIDxlbC1kZXNjcmlwdGlvbnMtaXRlbSB2LXRlc3Q+a29vcmlvb2thbWk8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgIDxlbC1kZXNjcmlwdGlvbnMtaXRlbSBsYWJlbD1cIlRlbGVwaG9uZVwiPjE4MTAwMDAwMDAwPC9lbC1kZXNjcmlwdGlvbnMtaXRlbT5cbiAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gbGFiZWw9XCJQbGFjZVwiPlN1emhvdTwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgPGVsLWRlc2NyaXB0aW9ucy1pdGVtIGxhYmVsPVwiUmVtYXJrc1wiPlxuICAgICAgPGVsLXRhZyBzaXplPVwic21hbGxcIj5TY2hvb2w8L2VsLXRhZz5cbiAgICA8L2VsLWRlc2NyaXB0aW9ucy1pdGVtPlxuICAgIDxlbC1kZXNjcmlwdGlvbnMtaXRlbSBsYWJlbD1cIkFkZHJlc3NcIlxuICAgICAgPk5vLjExODgsIFd1emhvbmcgQXZlbnVlLCBXdXpob25nIERpc3RyaWN0LCBTdXpob3UsIEppYW5nc3VcbiAgICAgIFByb3ZpbmNlPC9lbC1kZXNjcmlwdGlvbnMtaXRlbVxuICAgID5cbiAgICA8L0VsRGVzY3JpcHRpb25zPlxuPC90ZW1wbGF0ZT5cbiIsInNyYy9QbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctMTQyOTktZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiXG4gIH1cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJhbGxvd0pzXCI6IHRydWUsXG4gICAgXCJjaGVja0pzXCI6IHRydWUsXG4gICAgXCJqc3hcIjogXCJwcmVzZXJ2ZVwiLFxuICAgIFwidGFyZ2V0XCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwidHlwZXNcIjogW1wiZWxlbWVudC1wbHVzL2dsb2JhbC5kLnRzXCJdXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwic3JjL2VsZW1lbnQtcGx1cy5qcyI6ImltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcbmltcG9ydCBFbGVtZW50UGx1cyBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmxldCBpbnN0YWxsZWQgPSBmYWxzZVxuYXdhaXQgbG9hZFN0eWxlKClcblxuZXhwb3J0IGZ1bmN0aW9uIHNldHVwRWxlbWVudFBsdXMoKSB7XG4gIGlmIChpbnN0YWxsZWQpIHJldHVyblxuICBjb25zdCBpbnN0YW5jZSA9IGdldEN1cnJlbnRJbnN0YW5jZSgpXG4gIGluc3RhbmNlLmFwcENvbnRleHQuYXBwLnVzZShFbGVtZW50UGx1cylcbiAgaW5zdGFsbGVkID0gdHJ1ZVxufVxuXG5leHBvcnQgZnVuY3Rpb24gbG9hZFN0eWxlKCkge1xuICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgIGNvbnN0IGxpbmsgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaW5rJylcbiAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgIGxpbmsuaHJlZiA9ICdodHRwczovL3ByZXZpZXctMTQyOTktZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdsb2FkJywgcmVzb2x2ZSlcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gIH0pXG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctMTQyOTktZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

Reply:
Oh, I probably see what you mean. However, I think it's better to keep the current logic. 

After the relevant configuration according to the example you provided, the `label` and `content` will be rendered separately, which seems to be difficult to handle uniformly, and I think it may be better to trigger both and then leave the relevant logic to the user to handle by himself.

WDYT?